site/_data/i18n/authors.yaml

```diff
@@ -1,3 +1,13 @@
+josephmedley:
+  title:
+    en: 'Joseph Medley'
```

Comment:
```suggestion
joemedley:
  title:
    en: 'Joe Medley'
```

Reply:
I fixed this on a previous PR which you then closed. Please update the others, as I want this reference to be the same as on web.dev in case we move content between the two.